packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -327,6 +327,7 @@ export abstract class TaskRunner {
 		} catch (error) {
 			this.taskErrored(taskId, error);
 		}
+		this.resetIdleTimer();
```

Comment:
Actually jsut realized it would be safest to have this in a `finally`, since `taskErrored` does a network call and might fail as well
```suggestion
		} finally {
			this.resetIdleTimer();
		}
```